.github/workflows/connect-test.yml

```diff
@@ -38,14 +40,14 @@ jobs:
           cache-dependency-path: "**/yarn.lock"
       - run: sed -i "/\"node\"/d" package.json
       - run: yarn install
-      - run: yarn build:libs
       - run: yarn workspace @trezor/connect-iframe build
```

Comment:
Do we even need to build `connect-iframe`? It seems that I forget to upload it as artifact and tests still work